.npmignore

```diff
@@ -0,0 +1,10 @@
+.sass-cache/
```

Comment:
Thoughts on white listing here instead of black listing here? We can do this using the files attribute in our package.json file - https://docs.npmjs.com/files/package.json#files